src/pybind/mgr/dashboard/controllers/rbd_mirroring.py

```diff
@@ -574,3 +578,42 @@ def set(self, pool_name, peer_uuid, cluster_name=None, client_id=None,
             rbd.RBD().mirror_peer_set_attributes(ioctx, peer_uuid, attributes)
 
         _reset_view_cache()
+
+
+@UIRouter('/block/mirroring', Scope.RBD_MIRRORING)
+class RbdMirroringStatus(BaseController):
+    @EndpointDoc('Display RBD Mirroring Status')
+    @Endpoint()
+    @ReadPermission
+    def status(self):
+        status = {'available': True, 'message': None}
```

Comment:
I think we support Python 3.8 `TypedDict` imported from `typing_extensions`:
```py
try:
    from typing import TypedDict
except ImportError:
    from typing_extensions import TypedDict

class ModuleStatusGuardResponse(TypeDict):
    available: bool
    message: Optional[str]

class RbdMirroringStatus(BaseController):
    def status(self):
        ...
        return ModuleStatusGuardResponse(available=True)
        ...
        return ModuleStatusGuardResponse(available=False, message="RBD Mirroring is not configured")
```

Reply:
```
from typing_extensions import TypedDict
ModuleNotFoundError: No module named 'typing_extensions'
```

I got this exception.  and I saw somewhere in the comments that `# typing_extensions will not be available for the real mgr server`

Reply:
Weird... I think we were relying on those somewhere. Those are available in [our main distros](https://pkgs.org/search/?q=typing-extensions) (CentOS, Ubuntu, and OpenSUSE).